addon/components/nav-category.js

```diff
@@ -60,7 +60,7 @@ export default Component.extend({
       frostNavigation.dismiss()
       if (name !== active) { // click on another tab
         run.later(() => {
-          if (this.isDestroying || this.isDestoryed) {
+          if (this.isDestroying || this.isDestoryed || frostNavigation.isDestroying || frostNavigation.isDestroyed) {
```

Comment:
I guess when  I call `destroy()` it's only going into `isDestroying` and not `isDestroyed` or my test would have failed.